scenes/quests/story_quests/template/2_combat/combat.tscn

```diff
@@ -82,7 +83,7 @@ revealed = false
 next_scene = "uid://x3wm2ce0ax8i"
 item = SubResource("Resource_a51xm")
 collected_dialogue = ExtResource("2_3qn31")
-dialogue_title = &"well_done"
+dialogue_title = &""
```

Comment:
This change is wrong – this means that the same dialogue will play when you collect the collectible as when the scene starts. Again I can fix this.